portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.CommentSection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comment-page")
```

Comment:
This is named identically to the servlet above. Is that intended? Also, add javadoc please.

I'm not crazy about calling this "page" either. To me, that indicates that I'm going to get some html in the response. Simply "/comment" seems more appropriate.

Reply:
This Servlet apart of the "portfolio" project rather than the "database-portfolio" project. The former was the first half of the walkthrough which uses HTTP requests but stores the data on the backend. The latter stores the data on a database.